packages/components/navbar/src/NavbarItem/NavbarItem.tsx

```diff
@@ -83,6 +84,16 @@ function _NavbarItem(
     </Comp>
   );
 
+  if (!title) {
+    item = (
+      <div>
+        <Tooltip content={label} placement="bottom" showDelay={600} usePortal>
+          {item}
```

Comment:
This probably has been discussed before: Why do we make the tooltip part of the navbar item? The same behavior can be already achieved today by combining `Tooltip` and `NavbarItem` outside of f36.

Reply:
If we want to always have tooltips for these items I think it makes sense to compose them here.